tests/integration/conftest.py

```diff
@@ -31,10 +36,93 @@
         COLUMNS.tax_year.name,
         COLUMNS.employer_id.name,
     ],
-    # DATASETS.tax_1040.name: "todo",
+    DATASETS.tax_1040.name: [
+        COLUMNS.simulant_id.name,
+        COLUMNS.tax_year.name,
+    ],
 }
 
 
+@pytest.fixture(scope="session")
+def split_sample_data_dir(tmpdir_factory):
+    datasets = [
+        DatasetNames.CENSUS,
+        DatasetNames.ACS,
+        DatasetNames.CPS,
+        DatasetNames.SSA,
+        DatasetNames.TAXES_W2_1099,
+        DatasetNames.WIC,
+        DatasetNames.TAXES_1040,
+        DatasetNames.TAXES_DEPENDENTS,
+    ]
+    split_sample_data_dir = tmpdir_factory.mktemp("split_sample_data")
+    for dataset_name in datasets:
+        data = _load_sample_data(dataset_name)
+        # Split the sample dataset into two and save in tmpdir_factory
+        # We are spliting on household_id as a solution for how to keep households together
+        # for the tax 1040 dataset.
+        # We are special casing the SSA dataset because that is the only one without the
+        # household_id columns
+        outdir = split_sample_data_dir.mkdir(dataset_name)
+        if dataset_name == DatasetNames.SSA:
+            split_idx = int(len(data) / 2)
+            data[:split_idx].to_parquet(outdir / f"{dataset_name}_1.parquet")
+            data[split_idx:].to_parquet(outdir / f"{dataset_name}_2.parquet")
+        else:
+            split_year_mask = data[COLUMNS.household_id.name].isin(
+                list(data[COLUMNS.household_id.name].unique())[
+                    : (int(len(data[COLUMNS.household_id.name].unique()) / 2))
+                ]
+            )
+            data[split_year_mask].to_parquet(outdir / f"{dataset_name}_1.parquet")
+            data[~split_year_mask].to_parquet(outdir / f"{dataset_name}_2.parquet")
+
+    return Path(split_sample_data_dir)
+
+
+@pytest.fixture(scope="session")
+def split_sample_data_dir_state_edit(tmpdir_factory):
+    datasets = [
+        DatasetNames.CENSUS,
+        DatasetNames.ACS,
+        DatasetNames.CPS,
+        DatasetNames.SSA,
+        DatasetNames.TAXES_W2_1099,
+        DatasetNames.WIC,
+        DatasetNames.TAXES_1040,
+        DatasetNames.TAXES_DEPENDENTS,
+    ]
+    split_sample_data_dir_state_edit = tmpdir_factory.mktemp("split_sample_data_state_edit")
+    for dataset_name in datasets:
```

Comment:
I'm not necessarily recommending this now, but I do wonder if it would be faster and/or better to have this function just load the previously-generated split data, chagne the states, and then save out separately. That would certainly be DRYer.